internal/pkg/service/buffer/store/schema/schema_test.go

```diff
@@ -168,51 +168,51 @@ func TestSchema(t *testing.T) {
 		},
 		{
 			s.Slices().Opened().InFile(fileKey).Prefix(),
-			"slice/opened/123/my-receiver/my-export/2006-01-02T08:04:05.000Z/",
+			"slice/opened/00000123/my-receiver/my-export/2006-01-02T08:04:05.000Z/",
 		},
 		{
 			s.Slices().Opened().ByKey(sliceKey).Key(),
-			"slice/opened/123/my-receiver/my-export/2006-01-02T08:04:05.000Z/2006-01-02T09:04:05.000Z",
+			"slice/opened/00000123/my-receiver/my-export/2006-01-02T08:04:05.000Z/2006-01-02T09:04:05.000Z",
 		},
 		{
 			s.Slices().Closing().InFile(fileKey).Prefix(),
-			"slice/closing/123/my-receiver/my-export/2006-01-02T08:04:05.000Z/",
+			"slice/closing/00000123/my-receiver/my-export/2006-01-02T08:04:05.000Z/",
 		},
 		{
 			s.Slices().Closing().ByKey(sliceKey).Key(),
-			"slice/closing/123/my-receiver/my-export/2006-01-02T08:04:05.000Z/2006-01-02T09:04:05.000Z",
+			"slice/closing/00000123/my-receiver/my-export/2006-01-02T08:04:05.000Z/2006-01-02T09:04:05.000Z",
 		},
 		{
 			s.Slices().Closed().InFile(fileKey).Prefix(),
-			"slice/closed/123/my-receiver/my-export/2006-01-02T08:04:05.000Z/",
+			"slice/closed/00000123/my-receiver/my-export/2006-01-02T08:04:05.000Z/",
 		},
 		{
 			s.Slices().Closed().ByKey(sliceKey).Key(),
-			"slice/closed/123/my-receiver/my-export/2006-01-02T08:04:05.000Z/2006-01-02T09:04:05.000Z",
+			"slice/closed/00000123/my-receiver/my-export/2006-01-02T08:04:05.000Z/2006-01-02T09:04:05.000Z",
```

Comment:
Okrem toho som rovno pridal padding `0` aj na `ProjectID` (int), moze sa nam to zist ak by sme potrebovali iterovat nad vsetkymi `projects` a v buducnosti to bude tazke zmenit.

Reply:
Ine zmeny v scheme nie su, vid tento test.